File: Java_Practice/Java/string_vowels_consonants.java
//Count Vowels and Consonants
import java.util.Scanner;

public class string_vowels_consonants {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a string: ");
        String str = sc.nextLine();
        int vowels = 0, consonants = 0;

        for(char ch : str.toLowerCase().toCharArray()) {
            if(ch >= 'a' && ch <= 'z') {
                if("aeiou".indexOf(ch) != -1) vowels++;
                else consonants++;
            }
        }
        System.out.println("Vowels: " + vowels + "\nConsonants: " + consonants);
    }
}

File: Python/string_vowels_consonants.py
#Count Vowels and Consonants
str1 = input("Enter a string: ").lower()
vowels = consonants = 0

for ch in str1:
    if ch.isalpha():
        if ch in 'aeiou':
            vowels += 1
        else:
            consonants += 1

print("Vowels:", vowels)
print("Consonants:", consonants)
